node/client/src/lib.rs

```diff
@@ -259,63 +259,63 @@ impl sc_client_api::BlockBackend<Block> for Client {
 impl sc_client_api::StorageProvider<Block, crate::FullBackend> for Client {
 	fn storage(
 		&self,
-		id: &BlockId<Block>,
+		hash: &<Block as BlockT>::Hash,
```

Comment:
Where did it come from?

Reply:
After updating the dependencies we need to make our code compatible with new dependencies.